en/identity-server/next/docs/references/conditional-auth/api-reference.md

```diff
@@ -744,51 +757,93 @@ The HTTP POST function enables sending HTTP POST requests to specified endpoints
           <td>HTTP request headers to be included in the POST request (optional).</td>
         </tr>
+        <tr>
+          <td style="white-space: nowrap;"><code>authConfig</code></td>
+          <td>An object containing the necessary metadata to invoke the API. See [AuthConfig](#authconfig) for more information.</td>
+        </tr>
+        <tr>
+          <td style="white-space: nowrap;"><code>eventHandlers</code></td>
+          <td>The object that contains the callback functions, which are to be called based on the result of the GET request.<br />
+          Supported results are <code>onSuccess</code>, <code>onFail</code> and <code>onTimeout</code> which can
+              have their own optional callbacks as anonymous functions. For these callbacks, the [context](#context) and [data](#data) parameters are passed.
+          </td>
+        </tr> 
       </tbody>
     </table>
 
   - **Example**
 
   ``` js
+      var authConfig = {
+          type: "clientcredential",
+          properties: {
+              consumerKey: "clientId",
+              consumerSecret: "clientSecret",
+              tokenEndpoint: "https://token-endpoint.com/token"
+          }
+      };
+  
       function onLoginRequest(context) {
           httpPost('https://example.com/resource', {
               "email": "test@wso2.com"
           }, {
               "Authorization": "Bearer token",
               "Accept": "application/json"
-          }, {
+          }, authConfig, {
               onSuccess: function(context, data) {
-                  Log.info('httpPost call succeeded');
-                  context.selectedAcr = data.status;
+                  Log.info("Successfully invoked the external API.");
                   executeStep(1);
               },
               onFail: function(context, data) {
-                  Log.info('httpPost call failed');
-                  context.selectedAcr = 'FAILED';
+                  Log.info("Error occurred while invoking the external API.");
+                  executeStep(2);
+              },
+              onTimeout: function(context, data) {
+                  Log.info("Invoking external API timed out.");
                   executeStep(2);
               }
           });
       }
   ```
-  
+
 !!! note     
-    To restrict HTTP POST requests to certain domains, configure the `deployment.toml` file as follows:
+    To restrict HTTP GET requests to certain domains, and configure the HTTP connection settings for `httpGet` and `httpPost` functions in adaptive authentication scripts, update the `deployment.toml` file as follows:
 
-    ``` toml
+    ```toml
     [authentication.adaptive]
     http_function_allowed_domains = ["example.com", "api.example.org"]
+    
+    [authentication.adaptive.http_connections]
+    connection_timeout = 5000
+    
+    [authentication.adaptive.http_connections]
+    read_timeout = 5000
+    
+    [authentication.adaptive.http_connections]
+    request_timeout = 5000
```

Comment:
This doesn't include the configuration related to retry count

Reply:
Fixed with https://github.com/wso2/docs-is/pull/4772/commits/cf4fb92a77bd7856736c909b594f1ff84eb6c9a0